image/gimp_brush.ksy

```diff
@@ -0,0 +1,32 @@
+meta:
+  id: gimp_brush
+  title: GIMP (GNU Image Manipulation Program) brush version 2 file
+  file-extension: gbr
+  license: CC0-1.0
+  endian: be
+doc-ref: https://gitlab.gnome.org/GNOME/gimp/-/blob/master/devel-docs/gbr.txt
+seq:
+  - id: header_size
+    type: u4be
+  - id: version
+    type: u4be
```

Comment:
> what should they set this value to in order to be recognized in GIMP or other applications? How does GIMP treat this value - is it currently just ignored, or does it switch some features on/off or changes the expected byte layout?

As I anticipated, the source code of GIMP basically answers all these questions on these lines: [GNOME / **gimp** `/app/core/gimpbrush-load.c:201-237`](https://github.com/GNOME/gimp/blob/9b6d59f38aaef8c560e9fd2a19011da8e9dc6264/app/core/gimpbrush-load.c#L201-L237)

Apparently, only three values are possible - 1, 2 and 3. However, with the current implementation limited to the version 2 (and you also state it in `/meta/title`:

https://github.com/kaitai-io/kaitai_struct_formats/blob/db1486016fe04c546b589f1b07708099a69eb172/image/gimp_brush.ksy#L3

), it would make sense to make this even more explicit with `valid: 2`. But if you'd like to extend the specification a bit to support also versions 1 and 3 like GIMP does, it would be appreciated :)